src/Util/Casacore.cc

```diff
@@ -257,6 +263,15 @@ void NormalizeUnit(casacore::String& unit) {
     }
 }
 
+bool IsGildasUnit(const casacore::String& unit) {
+    // Check if unit is in format from GILDAS CLASS software:
+    //     '<unit> (T<type>)' where the <type> describes the temperature T.
+    // Also test for casacore::Unit::check changes to the unit name where ' ' and '*' are replaced with '.'.
+    // For example: "K (Ta*)" -->  "K.(Ta.)" in casacore.
+    std::regex gildas_regex("[ ]*[a-zA-Z]+[ .]+\\(T[a-zA-Z_]+[*.]*\\)[ ]*");
+    return std::regex_match(unit, gildas_regex);
```

Comment:
1. You can omit the `[]` around the space at the start and end.
2. I would define this regex as a const variable outside the function.

Reply:
Done, not sure why it is not showing this code as outdated.